tests/expr/fbinary/test-truediv.py

```diff
@@ -92,8 +93,17 @@ def test_div_integers(seed):
 
     DT = dt.Frame(x=src1, y=src2)
     RES = DT[:, f.x / f.y]
-    EXP = dt.Frame([None if src2[i] == 0 else src1[i] / src2[i]
-                   for i in range(n)])
+    EXP_list = []
+    for i in range(n):
+        if src1[i] == 0 and src2[i] == 0:
+    	    EXP_list.append(None)
+        elif src1[i] > 0 and src2[i] == 0:
+            EXP_list.append(math.inf)
+        elif src1[i] < 0 and src2[i] == 0:
+            EXP_list.append(-math.inf)
```

Comment:
This could be simplified to 
```
elif src2[i] == 0:
  EXP_list.append(math.copysign(math.inf, src1[i]))
```

Reply:
I would have written
```
if src2[i] == 0:
    EXP_list.append(+math.inf if src1[i] > 0 else
                    -math.inf if src1[i] < 0 else
                    None)
else:
    EXP_list.append(src1[i] / src2[i])
```
but then it's really up to whatever each person finds most readable.

Reply:
I will toss a coin for it. ;)